applications/api-cat/pom.xml

```diff
@@ -158,7 +158,7 @@
         <dependency>
```

Comment:
Såvidt jeg kan se så er alle endringene i denne commiten  en refaktorering for bedre struktur. Er det en korrekt antagelse av meg ?